api/inventorypb/agents.proto

```diff
@@ -464,6 +502,56 @@ message ChangeMySQLdExporterResponse {
   MySQLdExporter mysqld_exporter = 1;
 }
 
+// Add/Change RDSExporter
+
+message AddRDSExporterRequest {
+  // The pmm-agent identifier which runs this instance.
+  string pmm_agent_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Service identifier.
+  string service_id = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // RDS username for scraping metrics.
+  string username = 3 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // RDS password for scraping metrics.
+  string password = 4;
+  // Use TLS for database connections.
+  bool tls = 5;
+  // Skip TLS certificate and hostname validation.
+  bool tls_skip_verify = 6;
```

Comment:
We don't actually connect to the database in rds_exporter. We don't need those fields.